k2s/cmd/k2s/cmd/image/push.go

```diff
@@ -36,27 +36,21 @@ var (
 )
 
 func init() {
+	pushCmd.Flags().String(imageIdFlagName, "", "Image ID of the container image")
+	pushCmd.Flags().StringP(imageNameFlagName, "n", "", "Name of the container image including tag")
 	pushCmd.Flags().SortFlags = false
 	pushCmd.Flags().PrintDefaults()
 }
 
 func pushImage(cmd *cobra.Command, args []string) error {
```

Comment:
If the command is executed as **k2s image push**, without specifying any flags or arguments, the command prints out many errors. Possibly, because the script is invoked with empty name. Maybe flag or argument validation can be improved so that user is presented with an appropriate message.

Reply:
Noticed the same behavior for tag, and implemented a argument validation for both.